internal/mediators/config/registry.go

```diff
@@ -31,13 +32,22 @@ var registry = make(Registry)
 func GetOption(key string) Option {
 	rule, ok := registry[key]
 	if !ok {
-		return Option{key, String, EmptyEvent, EmptyEvent, false}
+		return Option{key, String, "", EmptyEvent, EmptyEvent, false}
 	}
 	return rule
 }
 
-func RegisterOption(key string, t Type, get Event, set Event) {
-	registry[key] = Option{key, t, get, set, true}
+// Registers a config option.
+// Name, Type, and Default are required fields.
+func RegisterOption(option Option) {
```

Comment:
I'm worried that policing the required fields via a docstring isn't enough. We should consider making the required options arguments to this function rather than just an `Option` argument.

Reply:
Okay, I've switched it back and created a separate function for registering with get/set events. Since all existing callers do not use get/set events, they are using the simplified version.